devdocs/process.md

```diff
@@ -131,12 +135,24 @@ Role: reviewer
     * The code is synced with upstream. GitHub should show it as 'can merge'. 
       If not, ask the dev to sync with upstream. 
   * If any of the above are not OK, 
+    * change the status of the PR to `s.Ongoing`
     * Add comments in the diff to suggest changes.
     * Optionally, add a comment in the conversation thread to inform the author to refine the code.
   * To remove white space changes from being shown, append `?w=1` to url of the `/files` page of the pull request (the "Files changed" tab)
   * If the code is OK on all aspects,
     * Change issue status to `s.ToMerge`
 
+Role: PM
+
+  * Review the code for maintainability and style
+  * Ensure appropriate header comments and expected standards are followed
+    * the standards used in TEAMMATES are available at [Readme](../README.md) under the *Supplementary documents* section
```

Comment:
These things are reviewer's job. PM does `a final check on the code quality and style`.